### DISCUSSION TITLE
The configuration scope of the keda parameter and The scope of use of the fallback parameter

### DISCUSSION THREAD

**Discussion by qingfenghcy** [kedacore/keda#3964] [Q&A / Need Help]
Hello， 
        By using and researching KEDA 2.8 and k8s 1.21, I currently have two questions。
        Firstly，the scaledobject yaml file have many configuration parameters， such as pollingInterval、stabilizationWindowSeconds. I would like to ask what is the upper and lower limit range for these parameters. 
        Secondly, If I use a prometheus custom metrics, whether the fallback parameter takes effect？If there is an anomaly in the prometheus metrics, what will happen to the production environment?
        Thank you very much! Looking forward to your response.

**Comment by zroubalik** [comment]
I don't have any specific data on the upper/lower limits, it depends on your usecase. 

Yeah, fallback works this way.
